app/helpers/setTheme.js

```diff
@@ -0,0 +1,28 @@
+import { helper } from '@ember/component/helper';
+import setCookie from './setCookie';
+import { SUN_URL, THEME } from '../constants/theme';
+
+function setTheme() {
+  const getCookies = document.cookie.split(';');
+  let cookies = {};
+
+  //storing cookie as key value pair in Object
+  getCookies.forEach((cookie) => {
+    const [key, value] = cookie.split('=');
```

Comment:
Maybe add an example of `cookie` in the comment for future understanding?
something like:
`// example of cookie: rds-session=ajkfhdfkahsfdfds`